src/ImageKit/Url/Url.php

```diff
@@ -201,36 +236,122 @@ private function constructTransformationString($transformation)
         for ($i = 0; $i < count($transformation); $i++) {
             $parsedTransformStep = [];
             foreach ($transformation[$i] as $key => $value) {
-                if (is_bool($value)) {
-                    $value = $value ? 'true' : 'false';
-                }
-                $value = (string)$value;
+                if($key=='if'){
```

Comment:
Please remove conditional transformation code till specification is discussed and agreeed.

Reply:
removed from code